css files/gethelpnow-style.css

```diff
@@ -22,6 +16,7 @@
 	src: url('../fonts/AvenirNextLTPro-Bold.otf');
 }
 
+/* ==PRIMARY LAYOUT STRUCTURE== */
```

Comment:
Please use single line comment instead of multiline. Also, don't use 'equal to' signs
For example, change this comment to: 
```
// Primary Layout Structure
```
Also, update other comments accordingly.